src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -76,7 +76,11 @@ exports.callCallbacks = function(queue : UpdateQueue, context : any) {
     if (callback && !node.callbackWasCalled) {
       node.callbackWasCalled = true;
       if (typeof context !== 'undefined') {
-        callback.call(context);
+        try {
+          callback.call(context);
+        } catch (error) {
+          // do something
```

Comment:
`trapError` is defined in `ReactFiberScheduler` and passed to `ReactFiberCommitWork` during instantiation. We can pass `trapError` function to `callCallbacks`

```js
callCallbacks(finishedWork.callbackList, instance, trapError);
```

Does this look good?